### PR TITLE
add metag wrapper doc and figures

### PR DESCRIPTION
**Pull request by kaijli**
did not add long reads or genomad because those are not implemented yet. unsure how much detail is needed.

### CODE REVIEW

**Comment by eecavanna** [comment]
Here's a link to a "rendered" view of the `.rst` file, in case anyone wants to review it from that perspective:
https://github.com/microbiomedata/docs/blob/14-metag-doc/content/nmdc/src/reference/metag_wrapper_doc.rst

File: content/nmdc/src/reference/metag_wrapper_doc.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The metagenome assembly workflow uses bbcms, metaSPAdes, and BBMap to run error correction, assembly, and assembly validation, respectively. The metagenome annotation workflow takes in assembled metagenomes and generates structural and functional annotations. The MAGs workflow uses metabat2 to generate metagenome bins and applies the MIMAG standards using annotated tRNAs, rRNAs, and marker genes with checkM to estimate completeness and contamination and subsequent taxonomic lineage assignment.
+
+Users can run a single workflow within the metagenome pipeline with the appropriate input files, but the entire metagenome workflow is available to run from start to finish on NMDC EDGE from a single input raw Illumina file.
+
+
+Workflow Availability
+---------------------
+The workflow is available in its individual components on GitHub and as a whole to run on `NMDC-EDGE <https://nmdc-edge.org/home>`_.
```
Comment (eecavanna) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:23]: I suggest removing the hyphen from the name of "NMDC EDGE". On the NMDC website, it is spelled without a hyphen.

image


```suggestion
The workflow is available in its individual components on GitHub and as a whole to run on `NMDC EDGE <https://nmdc-edge.org/home>`_. 
```

I have seen some team members spell it with a hyphen (in our day to day work). I'll create a ticket about this.

File: content/nmdc/src/reference/metag_wrapper_doc.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Hardware Requirements:
+~~~~~~~~~~~~~~~~~~~~~~
+- Disk space: 258 GB for databases 
+- 86 GB RAM
+
+Workflow Dependencies
+---------------------
+
+Third party software:
```
Comment (eecavanna) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:48]: In the sentence following this heading, "third-party" is hyphenated. I recommend hyphenating it here, too. I think hyphenation is grammatically correct in this case (citation goes here... 👀).

```suggestion
Third-party software:
```

File: content/nmdc/src/reference/metag_wrapper_doc.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+~~~~~~~~~~~~~~~~~~~~~
+
+Docker images containing third-party software are available in the repsective repositories. More information is available in the workflow repositories listed above.
```
Comment (eecavanna) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:51]: There was a typo in the word, "respective."

```suggestion
Docker images containing third-party software are available in the respective repositories. More information is available in the workflow repositories listed above. 
```

File: content/nmdc/src/reference/metag_wrapper_doc.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+#.	output file prefix
+#.  path to :code:`input_file` if interleaved file
+#.  paths to :code:`input_fq1` and :code:`input_fq2` non-interleaved paired-end reads 
+#.	input_interleaved (boolean)
+#.	RNA strand type (optional) either left blank, :code:`aRNA`, or :code:`non_stranded_RNA`
+
+
+Output:
+~~~~~~~
+Upon completion of the run, the NMDC EDGE interface provides users can view the results, which are grouped by individual workflow.
```
Comment (eecavanna) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:82]: I don't understand this phrase, grammatically:

> the NMDC EDGE interface provides users can view the results

File: content/nmdc/src/reference/metag_wrapper_doc.rst [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+~~~~~~~
+Upon completion of the run, the NMDC EDGE interface provides users can view the results, which are grouped by individual workflow. 
+
+In addition to the workflow outputs are summary tables for each portion: 
+
+- ReadsQC: statistics and metrics, including the number of reads and bases before and after QC filtering.
+- Read-based taxonomy: summary tables and interactive Krona plots as visual outputs.
+- Assembly: summary statistics
+- Annotation: statistics for processed sequences, predicted genes, and general quality information
+- MAGs: summary section with information on binned and unbinned contigs, genome completeness, estimated contamination, and the number of genes present on all bins determined to be high quality or medium quality.
```
Comment (eecavanna) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:90]: I noticed that some of these end with a period and some do not. I recommend either ending all of them with a period or not ending any of them with a period.

File: content/nmdc/src/reference/metag_wrapper_doc.rst
Comment (aclum) [inline content/nmdc/src/reference/metag_wrapper_doc.rst:1]: fix casing of metabat should be 'MetaBAT 2'
remove references to 'RNA strand type' in the input section

**Review by aclum — APPROVED** [review]

**Pull request merged**